lib/ParseSIL/ParseSIL.cpp

```diff
@@ -3119,24 +3119,20 @@ bool SILParser::parseSILInstruction(SILBuilder &B) {
   case SILInstructionKind::GraphOperationInst: {
     bool noClustering = false;
     if (P.consumeIf(tok::l_square)) {
+      // The only valid option is [no_clustering], when we see an l_square.
+      noClustering = true;
+
       Identifier ident;
       SourceLoc identLoc;
```

Comment:
`identLoc` is not being used after parsing, and it can be dropped. `parseSILIdentifier` is overloaded for both taking a `SourceLoc` and not taking one.

Reply:
Thanks for the pointer. Dropped it.